InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/VelocityDrive.java

```diff
@@ -25,7 +25,7 @@ public void execute() {
         double rawSpeed = DRIVE_SUBSYSTEM.getDriverRawSpeed();
         double rawTurn = DRIVE_SUBSYSTEM.getDriverRawTurn();
 
-        DRIVE_SUBSYSTEM.velocityDrive(rawSpeed, rawTurn);
+        DRIVE_SUBSYSTEM.drive(rawSpeed, rawTurn);
```

Comment:
I think we can delete this command now.

Reply:
It's used in AutoAlign still; but I think if I remove it then it would default to the drive() command anyways,  right?